pages/linux/lscpu.md

```diff
@@ -0,0 +1,15 @@
+# lscpu
+
+> Displays information about the CPU architecture.
+
+- Display information about all online CPUs:
+
+`lscpu`
+
+- Display information about offline CPUs:
+
+`lscpu --offline`
```

Comment:
This is what I get.
```
$lscpu --offline
lscpu: options --all, --online and --offline may only be used with options --extended or --parse.
```

Reply:
Oops! Looks like I made a slight mistake. Correcting now!